src/test/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContextPathVariableResolverTest.java

```diff
@@ -45,116 +45,143 @@ public class WebRequestProcessingContextPathVariableResolverTest {
 	@Parameters(name = "TestController: {0}")
 	public static Collection testController() {
 		return Arrays.asList(
-				TestControllerWithClassLevelRequestMappingWithValue.class,
-				TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class,
-				TestControllerWithClassLevelRequestMappingWithPath.class,
-				TestControllerWithClassLevelRequestMappingWithPathAndPathVarWithRegexp.class
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValue.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class, Boolean.TRUE },
```

Comment:
Please extract:
* `Boolean.TRUE` to constant: `REQUEST_ATTRIBUTES_WITH_PATH_VARIABLES`
* `Boolean.FALSE` to constant: `REQUEST_ATTRIBUTES_WITHOUT_PATH_VARIABLES`

Reply:
please explain in javadoc why we set true for one controller and false for another (even though they all seem to use path variables).